Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,31 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            for (int i = 0; i < 2; i++)
+            {
+                Console.WriteLine("Enter person " + (i + 1) + " first name:");
+                string fName = Console.ReadLine();
+
+                Console.WriteLine("Enter " + fName + "'s last name:");
+                string lName = Console.ReadLine();
+
+                Console.WriteLine("Enter " + fName + " " + lName + "'s age:");
+                int age = Convert.ToInt32(Console.ReadLine());
+
+                Console.WriteLine("Enter " + fName + " " + lName + "'s weight in kg:");
+                float weight = float.Parse(Console.ReadLine());
+
+                Console.WriteLine("Enter " + fName + " " + lName + "'s height in cm:");
+                float height = float.Parse(Console.ReadLine());
+
+                Console.WriteLine(fName + " " + lName + "is " + age + " years old, their weight is " + weight + " kg and their height is " + height + " cm.");
+
+                //float bmi = weight / (height * height * 1000);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4886017)